main.tf

```diff
@@ -175,13 +175,10 @@ resource "google_container_node_pool" "self" {
         enabled = gvnic.value
       }
     }
-    dynamic "guest_accelerator" {
-      for_each = lookup(each.value.node_config, "accelerator_count", 0) > 0 ? [1] : []
-      content {
-        type               = lookup(each.value, "accelerator_type", "")
-        count              = lookup(each.value, "accelerator_count", 0)
-        gpu_partition_size = lookup(each.value, "gpu_partition_size", null)
-      }
+    guest_accelerator {
```

Comment:
It needs the dynamic block.
Otherwise it will add this block to every node pool. Not desired